app.js

```diff
@@ -31,6 +31,31 @@ function initApp(options) {
     if(!app.conf.port) { app.conf.port = 8888; }
     if(!app.conf.interface) { app.conf.interface = '0.0.0.0'; }
     if(!app.conf.compression_level) { app.conf.compression_level = 3; }
+    if(app.conf.cors === undefined) { app.conf.cors = '*'; }
+    if(!app.conf.csp) {
+        app.conf.csp =
+            "default-src 'self'; object-src 'none'; media-src *; img-src *; style-src *; frame-ancestors 'self'";
+    }
+
+    // set outgoing proxy
+    if(app.conf.proxy) {
```

Comment:
We already have that as `proxy`, this is a forgotten addition. The basic reason is that in production our proxy handles both, so we don't really need a separation. In fact, I'd argue that separating them makes that less obvious.